pkg/cache/context.go

```diff
@@ -765,3 +767,62 @@ func (ctx *Context) SchedulerNodeEventHandler() func(obj interface{}) {
 	// this is not required in some tests
 	return nil
 }
+
+func createConfigMapSelector() (labels.Selector, error) {
+	slt := labels.NewSelector()
+	appVals := [] string {"yunikorn"}
+	req, err := labels.NewRequirement(common.LabelApp, selection.Equals, appVals)
+	if err != nil {
+		return  nil, err
+	}
+	slt = slt.Add(*req)
+	return slt, nil
+}
```

Comment:
Can you pls check if this can be simplified by

```
labels.SelectorFromSet()
```

if that can work, seems like we do not need a separate function for this.

Reply:
It works, I changed it